packages/react/src/auto/hooks/useFieldMetadata.tsx

```diff
@@ -0,0 +1,16 @@
+import { useFormFields } from "../AutoForm.js";
+import { useAutoFormMetadata } from "../AutoFormContext.js";
+
+export const useFieldMetadata = (fieldApiIdentifier: string) => {
```

Comment:
Could you add a test that checks we load the right field from the metadata? Check https://testing-library.com/docs/react-testing-library/api/#renderhook